.github/workflows/lint_code.yml

```diff
@@ -0,0 +1,36 @@
+name: Lint
+
+on:
+  push:
+
+env:
+  TERRAFORM_VERSION: "1.5.7"
+
+jobs:
+  linting:
+    name: Lint Terraform files
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Cancel previous runs
+        uses: styfle/cancel-workflow-action@0.11.0
+        with:
+          access_token: ${{ github.token }}
```

Comment:
We may not need this action by using the built-in [concurrency](https://docs.github.com/en/actions/using-jobs/using-concurrency) feature of Github action. Let's check if it is possible

Reply:
I changed it to use `concurrency` instead in ad618c7 ✅